src/live-programming.rst

```diff
@@ -16,6 +16,379 @@ REPLs
 
 Data Analysis Environments
 ==========================
+Data scientists use data analysis environments for a variety of data science activities.
+With the notion of extracting knowledge and insights from data, the term data science was first distinguished from pure statistics in 1997 :cite:`donoho:2017`. 
+University of Michigan statistics professor C.F. Jeff Wu popularized the term "data science" in his talk "Statistics = Data Science?".
+He described the current state of data science work as: 1) data collection, 2) data modeling and analysis, 3) problem solving and decision support.
+He argued that there should be a balanced curriculum that puts more emphasis on data collection, and there should be more focus on large and complex data which usually interface with other disciplines.
+
+.. container:: bib-item
+
+  .. bibliography:: data-analysis-environments.bib
+    :filter: key == 'donoho:2017'
+
+  This article reflects on the past 50 years of history of data science. It introduces how the field of data science emerged as a superset of statistics and machine learning, driven by commercial rather than intellectual developments. It also visions how the field of data science will grow in the next 50 years.
+
+Data science has grown rapidly over the last decade as datasets have increased rapidly in size and ubiquity. 
+Kery and Myers :cite:`kery:2017:exploring` observe that data science practices are a form of exploratory programming, with the results of previous steps determining the next steps that the programmer will take.
+Muller et al. :cite:`muller:2019` investigated how data science workers engaged with data.
+They examined the workflow of data science by interviewing 21 data science professionals with diverse backgrounds and classifying how they work with data.
```

Comment:
These kinds of details should be in the summary of the paper not in the introduction